build.sbt

```diff
@@ -47,6 +47,8 @@ lazy val commonSettings = Seq(
     "com.typesafe.akka"          %% "akka-protobuf-v3"        % "2.6.6",
     "com.typesafe.akka"          %% "akka-stream"             % "2.6.6",
     "com.typesafe.akka"          %% "akka-testkit"            % "2.6.6" % "test",
+    "com.typesafe.akka"          %% "akka-actor-typed"        % "2.6.6",
+    "com.typesafe.akka"          %% "akka-cluster-typed"      % "2.6.6",
```

Comment:
Oops didn't mean to include this. But I got some typed actors coming so we may as well.